server/src/main/java/com/cloud/ha/HighAvailabilityManagerImpl.java

```diff
@@ -844,29 +851,25 @@ public boolean configure(final String name, final Map<String, Object> xmlParams)
         value = params.get("force.ha");
         _forceHA = Boolean.parseBoolean(value);
 
-        value = params.get("time.to.sleep");
-        _timeToSleep = (long)NumbersUtil.parseInt(value, 60) * 1000;
+        _timeToSleep = TimeToSleep.value() * SECONDS_TO_MILLISECONDS_FACTOR;
 
-        value = params.get("max.retries");
-        _maxRetries = NumbersUtil.parseInt(value, 5);
+        _maxRetries = MaxRetries.value();
 
-        value = params.get("time.between.failures");
-        _timeBetweenFailures = NumbersUtil.parseLong(value, 3600) * 1000;
+        _timeBetweenFailures = TimeBetweenFailures.value() * SECONDS_TO_MILLISECONDS_FACTOR;
 
-        value = params.get("time.between.cleanup");
-        _timeBetweenCleanups = NumbersUtil.parseLong(value, 3600 * 24);
+        _timeBetweenCleanups = TimeBetweenCleanup.value();
 
         value = params.get("stop.retry.interval");
-        _stopRetryInterval = NumbersUtil.parseInt(value, 10 * 60);
+        _stopRetryInterval = NumbersUtil.parseInt(value, STOP_RETRY_INTERVAL_SECONDS);
 
         value = params.get("restart.retry.interval");
-        _restartRetryInterval = NumbersUtil.parseInt(value, 10 * 60);
+        _restartRetryInterval = NumbersUtil.parseInt(value, RESTART_RETRY_INTERVAL_SECONDS);
 
         value = params.get("investigate.retry.interval");
-        _investigateRetryInterval = NumbersUtil.parseInt(value, 1 * 60);
+        _investigateRetryInterval = NumbersUtil.parseInt(value, INVESTIGATE_RETRY_INTERVAL_SECONDS);
 
         value = params.get("migrate.retry.interval");
-        _migrateRetryInterval = NumbersUtil.parseInt(value, 2 * 60);
+        _migrateRetryInterval = NumbersUtil.parseInt(value, MIGRATE_RETRY_INTERVAL_SECONDS);
```

Comment:
Ah, sorry I missed this earlier but while we are exposing some config keys - perhaps we can also expose "migrate.retry.interval", "investigate.retry.interval", "restart.retry.interval", "stop.retry.interval", "force.ha" through ConfigKey?

Reply:
ok, I will also changed this deprecated key definitions.